screens/Menu/ShuttleBus.js

```diff
@@ -1,32 +1,245 @@
-import React from "react";
-import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
-import { Button } from "native-base";
-import { Feather } from "@expo/vector-icons";
+import React, { useEffect } from "react";
+import { View, Text, StyleSheet, Image, TouchableOpacity, FlatList, SafeAreaView } from "react-native";
+import { ListItem } from "react-native-elements";
+import { Feather, MaterialCommunityIcons } from "@expo/vector-icons";
 import PropTypes from "prop-types";
+import SegmentedControlTab from "react-native-segmented-control-tab";
+import { ShuttleBusTimes } from "../../constants/shuttleBustimes";
 
-/**Prop passed
+/**
+ * This a function is a mini component that returns the layout and the data of each active tab
+ * @param {*} props 
+ */
+function TimesToDisplay (props) {
+    let DATA = [];
+    let campus = props.campus;
+    if (props.campus == "SGW") {
+        DATA = props.data.sgwStops;
+    }
+    else if (props.campus == "Loyola") {
+        DATA = props.data.LoyolaStops;
+    }
+    return (
+        <SafeAreaView style={styles.SafeAreaViewStyle}>
+            <FlatList
+                data={DATA}
+                showsVerticalScrollIndicator={true}
+                keyExtractor={item => item.id}
+                renderItem={({ item }) => (
+                    <ListItem
+                        title={`Shuttle Bus to ${campus}`}
+                        titleStyle={styles.titleStyle}
+                        subtitle={`${item.hour}:${item.minutes}`}
+                        subtitleStyle={styles.subtitleStyle}
+                        bottomDivider
+                        containerStyle={styles.containerStyle}
+                        leftIcon={<MaterialCommunityIcons name="bus-clock" size={ICON_SIZE} style={styles.leftIcon} />}
+                        rightTitle={`${item.timeDifference.Hours}${item.timeDifference.Minutes}`}
+                        rightTitleStyle={styles.rightTitleStyle}
+                    />
+                )}
+            />
+        </SafeAreaView>
+    );
+}
+
+TimesToDisplay.propTypes = {
+    props: PropTypes.object,
+    campus: PropTypes.any,
+    data: PropTypes.any
+};
+
+const ICON_SIZE = 35;
+
+/** Prop passed
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
-function ShuttleBus(props) {
+function ShuttleBus (props) {
+    const [selectedTab, setSelectedTab] = React.useState(0);
+    const [currentTime, setCurrentTime] = React.useState(null);
+    const [currentDay, setCurrentDay] = React.useState(null);
+    const [campus, setCampus] = React.useState("SGW");
+    const [results, setResults] = React.useState([]);
 
-     /**
-     * The method will slide the side menu from the right side of the screen
-     * @param  {} =>{props.navigation.openDrawer(
-     */
+    const weekDays = ["sunday", "monday", "tuesday", "wednesday", "thursday", "saturday", "sunday"];
```

Comment:
This is no longer accurate since the index at 6 is Sunday but it should be Saturday... I suggest removing the last element.

Reply:
you're absolutely right, good catch!!